src/web/index.html

```diff
@@ -14,161 +18,233 @@
     <script src="./js/ass.min.js" defer></script>
     <script src="./js/sql.min.js" defer></script>
     <script src="./js/exif-js.min.js" defer></script>
+    <link rel="stylesheet" href="./css/materialdesignicons/mdi.css" />
```

Comment:
Use this directly in the theme of the new frontend, the index should not apply any theme based styles